src/CFBG.cpp

```diff
@@ -145,7 +144,24 @@ TeamId CFBG::GetLowerSumPlayerLvlTeamInBg(Battleground* bg)
         return GetLowerAvgIlvlTeamInBg(bg);
     }
 
-    return (playerLevelAlliance < playerLevelHorde) ? TEAM_ALLIANCE : TEAM_HORDE;
+    TeamId lowestTeam = (playerLevelAlliance < playerLevelHorde) ? TEAM_ALLIANCE : TEAM_HORDE;
+
+    if (IsEnableEvenTeams())
+    {
+        if (joiningPlayers % 2 == 0)
+        {
+            // if who is joining has the level (or avg item level) lower than the average players level of the joining-queue, so who actually can enter in the battle
+            // put him in the stronger team, so swap the lowestTeam
+            if (player->getLevel() <  averagePlayersLevelQueue || (player->getLevel() == averagePlayersLevelQueue && player->GetAverageItemLevel() < averagePlayersItemLevelQueue))
+            {
+                lowestTeam = lowestTeam == TEAM_ALLIANCE ? TEAM_HORDE : TEAM_ALLIANCE;
+            }
+        }
+
+        joiningPlayers--;
+    }
+
+    return lowestTeam;
```

Comment:
if I understood correctly:

> put him in the stronger team, so swap the lowestTeam

then the `lowestTeam` will not be actually the "lowest" team anymore (as the weakest team). So why don't we rename this variable to something like `selectedTeam`? the same for the method name